packages/bsky/src/services/feed/views.ts

```diff
@@ -342,4 +375,39 @@ export class FeedViews {
       media: mediaEmbed,
     }
   }
+
+  formatGate(gate: GateRecord, lists: ListInfoMap) {
```

Comment:
Do we want to include the original record or uri of the gate here?

Reply:
hmm the uri is deterministic based on post uri so not necessary, but does seem kinda nice....

in general, i like the idea of including primary content where it doesn't add a bunch of overhead

maybe a gateview could look something like: 
```
{
  record: Record
  lists: ListView[]
}
```
or
```
{
  record: Record
  view: #gateView
}
```
the latter is similar to how we do post embeds, but is more duplicate data

Reply:
Ended-up going with the former, seems like it should be just fine here.  Also allowed me to avoid needing to present not-founds.  I added in `{ uri, cid, record }` since that seems like it might be sort of standard-issue record info.